### PR TITLE
Lesson 3 exercises

### PR DESCRIPTION
**Pull request by nam-vo**

### CODE REVIEW

File: students/nam_vo/lesson03/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    print(l)
+
+    # Ask the user for another fruit and add it to the end of the list
+    fruit = input('Enter another fruit > ')
+    l.append(fruit)
+    # Display the list
+    print(l)
+
+    # Ask the user for a number and display the number back to the user and the fruit corresponding to that number (on a 1-is-first basis)
+    number = input('Enter a number > ')
```
Comment (charlesrobison) [inline students/nam_vo/lesson03/list_lab.py]: Recommend using len() here to let the user know how long the list is

File: students/nam_vo/lesson03/list_lab.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    l = ['Watermelon'] + l
+    print(l)
+
+    # Add another fruit to the beginning of the list using insert() and display the list
+    l.insert(0, 'Kiwi')
+    print(l)
+
+    # Display all the fruits that begin with “P”, using a for loop
+    for item in l:
+        if item[0] == 'P':
```
Comment (charlesrobison) [inline students/nam_vo/lesson03/list_lab.py]: You can use upper() to catch any case sensitive user input

File: students/nam_vo/lesson03/list_lab.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+def series2(a_list):
+    # Display the list
+    print(a_list)
+
+    # Remove the last fruit from the list and display the list
+    a_list.pop()
+    print(a_list)
+
+    # Ask the user for a fruit to delete, find it and delete it
+    fruit = input('Enter a fruit name > ')
```
Comment (charlesrobison) [inline students/nam_vo/lesson03/list_lab.py]: Might want to say 'Enter a fruit name to remove it from the list' or similar.  More explicit

File: students/nam_vo/lesson03/list_lab.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    for fruit in a_list[:]:
+        answer = input(f'Do you like {fruit.lower()}?(yes/no) > ')
+        # For any answer that is not “yes” or “no”, prompt the user to answer with one of those two values
+        while not (answer == 'yes' or answer == 'no'):
+            answer = input(f'Do you like {fruit.lower()}?(yes/no) > ')
+        # For each “no”, delete that fruit from the list
+        if answer == 'no':
+            a_list.remove(fruit)
+    # Display the list
+    print(a_list)
```
Comment (charlesrobison) [inline students/nam_vo/lesson03/list_lab.py]: Nice!

File: students/nam_vo/lesson03/slicing_lab.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    every_other_seq = last_4_removed_seq[::2]
+    return every_other_seq
+
+def reverse_element(seq):
+    """Return a copy of seq with the elements reversed (just with slicing)"""
+    return seq[::-1]
+
+def slice_third(seq):
+    """Return a copy of seq with the last third, then first third, then the middle third in the new order"""
+    return (seq[-3:-2] + seq[2:3] + seq[5:6])
```
Comment (natasha-aleksandrova) [inline students/nam_vo/lesson03/slicing_lab.py]: this is incorrect - if your input is (1, 2, 3, 4, 5, 6, 7, 8, 9) then expected output is: (7, 8, 9, 1, 2, 3, 4, 5, 6)

**Pull request merged**